src/app/app-routing.module.ts

```diff
@@ -0,0 +1,34 @@
+import {NgModule} from '@angular/core';
+import {RouterModule, Routes} from '@angular/router';
+import {HOME_PAGE, LOGIN_PAGE, LOGOUT_PAGE} from './constants/common';
+import {UnauthGuard} from './shared/guard/unauth.guard';
+import {NotFoundComponent} from './core/pages/not-found/not-found.component';
+
+const routes: Routes = [
+  {path: '', redirectTo: LOGIN_PAGE, pathMatch: 'full'},
+  {
+    path: LOGOUT_PAGE,
+    loadChildren: () =>
+      import('src/app/auth/auth.module').then((m) => m.AuthModule),
+  },
+  {
+    path: LOGIN_PAGE,
+    loadChildren: () =>
+      import('src/app/auth/auth.module').then((m) => m.AuthModule),
+  },
```

Comment:
the same lazy  module is used for different routes , it would be better to join this routes into one, or split AuthModule